Create AWS DataCenter Server broken when specifying sts_external_id

When attempting to create an AWS DataCenter Server object, the call made by the provider contains an invalid key "custom-value"

```
Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # module.global.checkpoint_management_aws_data_center_server.dc_test_name will be created
  + resource "checkpoint_management_aws_data_center_server" "dc_test_name" {
      + authentication_method  = "role-authentication"
      + color                  = "black"
      + enable_sts_assume_role = true
      + id                     = (known after apply)
      + ignore_errors          = false
      + ignore_warnings        = false
      + name                   = "dc-test-name"
      + region                 = "us-east-1"
      + sts_external_id        = "id-string"
      + sts_role               = "test-role"
    }

Plan: 1 to add, 0 to change, 0 to destroy.
module.global.checkpoint_management_aws_data_center_server.dc_test_name : Creating...
module.global.checkpoint_management_aws_data_center_server.dc_test_name : Still creating... [10s elapsed]
╷
│ Error: fail to add-data-center-server. task-id [01[234](https://github.com/actions/runs/4294243808/jobs/7483005056#step:13:235)567-89ab-cdef-8d17-2f261c3f10e8]
│ Message: Validation failed, unrecognized parameter: [custom-value].
│ 
│   with module.global.checkpoint_management_aws_data_center_server.dc_test_name,
│   on global/objects-datacenter.tf line 1, in resource "checkpoint_management_aws_data_center_server" "dc_test_name":
│    1: resource "checkpoint_management_aws_data_center_server" "dc_test_name" {
│ 
╵
Error: Terraform exited with code 1.
Error: Process completed with exit code 1.
```

--- a/checkpoint/resource_checkpoint_management_aws_data_center_server.go
+++ b/checkpoint/resource_checkpoint_management_aws_data_center_server.go
@@ -134,7 +134,7 @@ func createManagementAwsDataCenterServer(d *schema.ResourceData, m interface{}) 
 	}
 
 	if v, ok := d.GetOk("sts_external_id"); ok {
-		awsDataCenterServer["custom-value"] = v.(string)
+		awsDataCenterServer["sts-external-id"] = v.(string)
 	}
 
 	if v, ok := d.GetOk("tags"); ok {

